doc/templates/uno/component/navbar.scss

```diff
@@ -119,13 +115,28 @@
         font-size: 19px;
         line-height: 17px;
       }
+
+      // Media queries for font size and padding adjustments
+      @media (min-width: 1024px) {
+        font-size: 14px;
+        line-height: 17px;
+        padding: 0.2rem !important;
+      }
+
+      @media (min-width: 1280px) {
+        font-size: 16px;
+        line-height: 17px;
+        padding: 0.2rem !important;
+      }
+
+      @media (min-width: 1366px) {
+        font-size: 19px;
+        line-height: 17px;
+        padding: 0.2rem !important;
+      }
     }
 
-    /**
-     * Top level menu items
-     */
```

Comment:
@marcoscodnet why are we removing all these previous comments here? Those should be kept in my opinion to give more context and help when coming back to this code.

Reply:
![image](https://github.com/user-attachments/assets/a8e926ac-939e-4cc7-996f-a2b24297311a)